.repo-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "dashboard",
```

Comment:
Looks like we're mixing `dashboards` and `monitoring dashboards` in a few places.  Can we standardize on `monitoring-dashboards`?